src/components/shared/prompt/prompt.tsx

```diff
@@ -0,0 +1,58 @@
+import { Modal } from "@lifesg/react-design-system/modal";
+import { TestHelper } from "../../../utils";
+import {
+	ButtonContainer,
+	Container,
+	Description,
+	LabelContainer,
+	PromptButton,
+	PromptImage,
+	Title,
+} from "./prompt.styles";
+import { IPromptProps } from "./types";
+
```

Comment:
`Prompt` is a modal that comes with buttons, it is meant to be used for confirmations